octobot_tentacles_manager/managers/tentacles_setup_manager.py

```diff
@@ -0,0 +1,139 @@
+#  Drakkar-Software OctoBot-Tentacles-Manager
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+from asyncio import gather
+from os.path import join, isfile, exists
+from shutil import rmtree
+
+import aiofiles
+
+from octobot_commons.logging.logging_util import get_logger
+from octobot_tentacles_manager.configuration.tentacles_setup_configuration import TentaclesSetupConfiguration
+from octobot_tentacles_manager.constants import USER_TENTACLE_SPECIFIC_CONFIG_PATH, PYTHON_INIT_FILE, \
+    TENTACLES_FOLDERS_ARCH, DEFAULT_TENTACLE_CONFIG, TENTACLES_PATH, USER_TENTACLE_CONFIG_PATH, \
+    TENTACLES_REQUIREMENTS_INSTALL_TEMP_DIR
+from octobot_tentacles_manager.util.file_util import find_or_create
+from octobot_tentacles_manager.util.tentacle_explorer import load_tentacle_with_metadata
+
+
+class TentaclesSetupManager:
+
+    TENTACLE_IMPORT_HEADER = """from octobot_tentacles_manager.api.inspector import check_tentacle_version
+from octobot_commons.logging.logging_util import get_logger
+"""
+
+    def __init__(self, tentacle_path):
+        self.tentacle_path = tentacle_path
+        self.default_tentacle_config = DEFAULT_TENTACLE_CONFIG
+
+    async def refresh_tentacles_config_file(self):
```

Comment:
This should be done in a Tentacle specific class

Reply:
I renamed this methods, it's about updating the global tentacles config file, it's relative to the global tentacle setup, not one tentacle in particular.